src/test/java/com/cloudbees/jenkins/support/filter/ContentMappingsTest.java

```diff
@@ -25,17 +25,24 @@
 
 import hudson.model.FreeStyleProject;
 import jenkins.model.Jenkins;
+import org.apache.commons.io.IOUtils;
```

Comment:
Do you need any of these imports anymore?

Reply:
No, I don't, Thanks Matt.